www/src/components/blog/avatarList.astro

```diff
@@ -18,7 +18,7 @@ type Commit = {
 async function getCommits(url: string) {
   try {
     const token = import.meta.env.SNOWPACK_PUBLIC_GITHUB_TOKEN ?? "hello";
-    if (!token) {
+    if (!token || token === "hello") {
```

Comment:
I think we should probably remove the `?? 'hello'` instead?

Reply:
yeah  i agree